config/routes.rb

```diff
@@ -77,6 +77,12 @@
 
       end
 
+      resource :role_list, only: [:destroy, :create, :new] do
+        get 'deletable' => 'role_lists#deletable'
+        get 'move'      => 'role_lists#move'
+        get 'movable'  => 'role_lists#movable'
```

Comment:
:rage: 

```suggestion
        get 'movable'   => 'role_lists#movable'
```

:smile: